README.md

```diff
@@ -1,49 +1,38 @@
 ## NYPL ResearchNow
 
 ### Version
-> 0.9.3
+> 0.10.1
 
 ### ResearchNow Search & Retrieval Application
 [![GitHub version](https://badge.fury.io/gh/NYPL%2Fsfr-bookfinder-front-end.svg)](https://badge.fury.io/gh/NYPL%2Fsfr-bookfinder-front-end)
-[![Build Status](https://travis-ci.com/NYPL/sfr-bookfinder-front-end.svg?branch=development)](https://travis-ci.com/NYPL/sfr-bookfinder-front-end)
-[![Dependencies Status](https://david-dm.org/NYPL/sfr-bookfinder-front-end/status.svg)](https://david-dm.org/NYPL/sfr-bookfinder-front-end)
-[![devDependencies Status](https://david-dm.org/NYPL/sfr-bookfinder-front-end/dev-status.svg)](https://david-dm.org/NYPL/sfr-bookfinder-front-end?type=dev)
 
-ResearchNow front end application based on NYPL's React Design System.
+Digital Research Books' front end application based on NYPL's React Design System.
 
-Provides a "Welcome page" entry point with heading, search box, and tagline. Connects to an ElasticSearch index via an API endpoint (https://platform.nypl.org/research-now/search-api).
-Simple keyword searches can be entered in the form and an index response is displayed back to
-the user.
-
-### Querying
+Provides a "Welcome page" entry point with heading, search box, and tagline. Connects to an ElasticSearch index via an API endpoint (https://drb-api-production.nypl.org/apidocs/).
```

Comment:
I would make the API endpoint https://digital-research-books-api.nypl.org here just so it's consistent with the URL we generally publicize.